.github/workflows/lint.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
       - uses: cachix/install-nix-action@v22
         with:
-          nix_path: nixpkgs=https://github.com/NixOS/nixpkgs/archive/43862987c3cf2554a542c6dd81f5f37435eb1423.tar.gz # keep in sync with shell.nix
+          nix_path: nixpkgs=https://github.com/NixOS/nixpkgs/archive/880992dcc006a5e00dd0591446fdf723e6a51a64.tar.gz # keep in sync with shell.nix
```

Comment:
A bit random to bump this in this commit.

Reply:
Yeah, but the newer version of nixpkgs is needed for `wait4x`...